tests/integration/cloud/rest_contracts/test_expectation_suites.py

```diff
@@ -363,68 +332,3 @@ def test_put_non_existent_expectation_suite(
     run_rest_api_pact_test: Callable[[ContractInteraction], None],
 ) -> None:
     run_rest_api_pact_test(contract_interaction)
-
-
-@pytest.mark.cloud
-@pytest.mark.skip(reason="unexpected 401 instead of 204 in CI only")
-def test_delete_expectation_suite(
-    pact_test: pact.Pact,
-    cloud_data_context: CloudDataContext,
-) -> None:
-    provider_state = "the Expectation Suite does exist"
-    scenario = "a request to delete an Expectation Suite"
-    method = "DELETE"
-    path = f"/organizations/{EXISTING_ORGANIZATION_ID}/expectation-suites"
-    query = {
-        "name": "brand new suite",
-    }
-    status = 204
-
-    (
-        pact_test.given(provider_state=provider_state)
-        .upon_receiving(scenario=scenario)
-        .with_request(
-            method=method,
-            path=path,
-            query=query,
-        )
-        .will_respond_with(
-            status=status,
-        )
-    )
-
-    with pact_test:
-        cloud_data_context.delete_expectation_suite(expectation_suite_name=query["name"])
-
-
-@pytest.mark.cloud
-@pytest.mark.skip(reason="unexpected 401 instead of 404 in CI only")
-def test_delete_non_existent_expectation_suite(
-    pact_test: pact.Pact,
-    cloud_data_context: CloudDataContext,
-) -> None:
-    provider_state = "the Expectation Suite does not exist"
-    scenario = "a request to delete an Expectation Suite"
-    method = "DELETE"
-    path = f"/organizations/{EXISTING_ORGANIZATION_ID}/expectation-suites"
-    query = {
-        "name": str(uuid.uuid4()),
-    }
-    status = 404
-
-    (
-        pact_test.given(provider_state=provider_state)
-        .upon_receiving(scenario=scenario)
-        .with_request(
-            method=method,
-            path=path,
-            query=query,
-        )
-        .will_respond_with(
-            status=status,
-        )
-    )
-
-    with pact_test:
-        with pytest.raises(DataContextError):
-            cloud_data_context.delete_expectation_suite(expectation_suite_name=query["name"])
```

Comment:
these tests do not work with the V1 API (suites factory), which requires the suite object in order to delete. There's conversation about changing that to name/ID. If that change occurs, these tests can be added back.